micronaut-camunda-bpm-feature/build.gradle

```diff
@@ -131,3 +132,29 @@ signing {
 configurations {
     testArtifacts
 }
+
+// When spotbugs is executed locally, it will fail when there is one finding
+// When spotbugs is executed in "continous integration" it will not fail and we can upload the spotbug results to PR
+boolean ignoreFailuresProperties = project.hasProperty('spotbugs.ignoreFailures') ? project.getProperty('spotbugs.ignoreFailures') == 'true' : true
```

Comment:
* do we need the comparison with "true"?
* it seems ignoreFailures is never set to false